crates/matrix-sdk-base/src/client.rs

```diff
@@ -704,7 +704,20 @@ impl BaseClient {
                                 room_info.base_info.is_marked_unread = event.content.unread;
                             });
                         }
+                        AnyRoomAccountDataEvent::UnstableMarkedUnread(event) => {
```

Comment:
We handle both the stable and unstable version to keep the old behavior.

Reply:
Maybe we could extract this to a function, so we don't forget to update one branch in the future?

Reply:
Done